src/messages.cpp

```diff
@@ -214,6 +216,32 @@ void Messages::deserialize( JsonObject &json )
     obj.read( "curmes", player_messages.impl_->curmes );
 }
 
+std::string Messages::replace_text( std::string& str, const std::string& from, const std::string& to )
+{
+    size_t start_pos = str.find(from);
+    if(start_pos == std::string::npos)
```

Comment:
Please follow the code style guide (see doc/CODE_STYLE.md). In this case: spaces after `(` and before `)` and brackets around the following block.